bundler/lib/bundler.rb

```diff
@@ -40,6 +40,7 @@ module Bundler
   SUDO_MUTEX = Thread::Mutex.new
 
   autoload :Checksum,               File.expand_path("bundler/checksum", __dir__)
+  autoload :CLI,                    File.expand_path("bundler/cli", __dir__)
```

Comment:
I have a hunch this was intentionally excluded 😨 

An alternative to this would be adding `require "bundler/cli"` down in `auto_install`

Reply:
@deivid-rodriguez do you have an opinion either way?

- a) add `CLI` to the list of autoloaded consts
- b) `require 'bundler/cli'` when it's needed during the autoinstall

Reply:
I think I'm good with `autoload` 👍.

Reply:
Awesome, sounds good. Marking this comment resolved, but let me know if there's anything else needed to get in shape to merge!